lib/components/Editor/CustomExtensions/Image/LocalUploader.jsx

```diff
@@ -46,9 +40,15 @@ const LocalUploader = ({
         return true;
       },
     })
-      .use(XHRUpload, { endpoint, ...UPPY_UPLOAD_CONFIG })
+      .use(ActiveStorageUpload, {
+        directUploadUrl: DIRECT_UPLOAD_ENDPOINT,
+        ...UPPY_UPLOAD_CONFIG,
+      })
       .on("upload", () => setIsUploading(true))
-      .on("upload-success", (_, response) => onSuccess(response.body.imageURL))
+      .on("upload-success", ({ data: { name } }, { signed_id }) => {
+        const imageUrl = `${window.location.origin}/rails/active_storage/blobs/redirect/${signed_id}/${name}`;
+        onSuccess(imageUrl);
```

Comment:
@sandip-mane _a I need your expertise in this issue. Could you please take a look at it and provide your thoughts?
https://vimeo.com/744279451/12227d0cac

Reply:
@AbhayVAshokan https://app.usebubbles.com/ah2eZDQftYg8LxUAgo6Tzo/file-path

Reply:
Oh yes. Great idea. Let me make the required changes.